assets/src/globals.js

```diff
@@ -7,7 +7,7 @@ const mylaGlobals = mylaGlobalsEl ? JSON.parse(mylaGlobalsEl.textContent) : {}
 
 let cspNonce
 const cspMetaEl = document.querySelector('meta[name="csp-nonce"]')
-if (cspMetaEl !== null & cspMetaEl.hasAttribute('content')) {
+if (cspMetaEl !== null && cspMetaEl.hasAttribute('content')) {
```

Comment:
Undo the change here as I don't feel this needed any change

Reply:
According to UM-GPT the single & causes some kind of error in a test suite where something in this line `if (cspMetaEl !== null & cspMetaEl.hasAttribute('content'))` is null

Reply:
OK, I think I missed seeing double `&&` and concluded as space change. This change is fine